packages/models/src/eservice/protobufConverterToV2.ts

```diff
@@ -119,12 +119,7 @@ export const toDescriptorV2 = (input: Descriptor): EServiceDescriptorV2 => ({
   archivedAt: dateToBigInt(input.archivedAt),
   rejectionReasons:
     input.rejectionReasons?.map(toDescriptorRejectedReasonV2) ?? [],
-  templateVersionRef: input.templateVersionRef
-    ? {
-        id: input.templateVersionRef.id,
-        interfaceMetadata: input.templateVersionRef.interfaceMetadata,
-      }
-    : undefined,
+  templateVersionRef: input.templateVersionRef,
```

Comment:
No need for that as well, we already spread `...input` above, in these methods we explicitly add a converter only when an override is needed 

```suggestion
```